src/main/java/com/slobodianyk/java_spring_boot/exception/TodoNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.slobodianyk.java_spring_boot.exception;
+
+public class TodoNotFoundException extends RuntimeException {
```

Comment:
No need to create a specific error for each entity. What if you'll have more entities: Car, User, Board, etc... Better use one generic exception like `EntityNotFoundException`